modules/stochastic_tools/test/tests/samplers/AdaptiveImportanceSampler/ais.i

```diff
@@ -0,0 +1,87 @@
+[StochasticTools]
+[]
+
+[Distributions]
+  [mu1]
+    type = Normal
+    mean = 0.0
+    standard_deviation = 0.5
+  []
+  [mu2]
+    type = Normal
+    mean = 1
+    standard_deviation = 0.5
+  []
+[]
+
+[Samplers]
+  [sample]
+    type = AdaptiveImportance
+    distributions = 'mu1 mu2'
+    execute_on = 'PRE_MULTIAPP_SETUP'
+    proposal_std = '0.15 0.15'
+    output_limit = 0.45
+    num_samples_train = 5
+    std_factor = 0.8
+    use_absolute_value = true
+    seed = 1012
+    initial_values = '-0.10329808102501603 1.2396280668056123'
+    inputs_reporter = 'adaptive_MC/inputs'
+  []
+[]
+
+[MultiApps]
+  [sub]
+    type = SamplerFullSolveMultiApp
+    input_files = ../../multiapps/sampler_full_solve_multiapp/sub.i
```

Comment:
You should just copy and paste the sub file instead of using one that is outside of this directory.